frontend/e2e/navigate-to-publishing-page-and-test-release.spec.ts

```diff
@@ -1,5 +1,14 @@
 import { test, expect, Page } from "@playwright/test"
 import { readFileSync } from "fs"
+import resetDatabase from "@e2e/database/resetDatabase"
+
+test.beforeAll(async () => {
```

Comment:
Todo: Is this something we expect to want to do in all test suites? If so, please move this to the [global setup and teardown](https://playwright.dev/docs/test-global-setup-teardown).